quench_resetter.py

```diff
@@ -1,53 +1,44 @@
-import logging
 from time import sleep
 
 from epics import PV
 from numpy.linalg import LinAlgError
 
 from lcls_tools.common.controls.pyepics.utils import PVInvalidError
-from lcls_tools.superconducting.sc_linac import Cryomodule
 from lcls_tools.superconducting.sc_linac_utils import (
     ALL_CRYOMODULES,
-    CavityFaultError,
     HW_MODE_ONLINE_VALUE,
+    CavityFaultError,
 )
-from quench_linac import QUENCH_MACHINE
+from quench_linac import QUENCH_MACHINE, QuenchCryomodule
```

Comment:
Is this class only called when a cavity queches? Not a huge deal, but maybe rename to QuenchedCryomodule/QuenchedCavity?

Reply:
No it's just the name of the cryomodule class used for this application